HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -315,14 +319,59 @@ wait_server(){
 	echo "server is ready!"
 }
 
+edit_model(){
+    cp -r $1 $TMPDIR/local_model_repo/
```

Comment:
to make the code a little easier to read/follow, I would start out by assigning `$1`, `$2`, etc. to local variables and then use the local variables everywhere.

Reply:
Done